janus_server/src/binary_utils/job_driver.rs

```diff
@@ -195,12 +199,12 @@ where
                         .await
                         {
                             Ok(Ok(_)) => debug!("Job stepped"),
-                            Ok(Err(err)) => {
-                                error!(?err, "Couldn't step job");
+                            Ok(Err(error)) => {
+                                error!(%error, "Couldn't step job");
                                 status = "error"
                             }
-                            Err(err) => {
-                                error!(?err, "Stepping job timed out");
+                            Err(_err) => {
+                                error!("Stepping job timed out");
```

Comment:
Note that the error here is of type `tokio::time::error::Elapsed`. This doesn't provide any information that isn't conveyed in the message, so I dropped it. Only Debug is implemented on Elapsed, so this would have just printed `Elapsed(())` previously.